### PR TITLE
fix(reactHtml): renderScripts adding unknown integrity

### PR DESCRIPTION
**Pull request by JAdshead**
## Description

Fallback externals do not support legacy, remove legacy logic to ensure does not load missing scripts.
Don't attempt to include integrity when missing, this is requires as fallback externals do not include integrity values

## Motivation and Context

Fix SSR when fallback externals are required and has no integrity value provided

## How Has This Been Tested?

Test suite

## Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [ ] Dependency update
- [ ] Security update

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

### CODE REVIEW

**Comment by Matthew-Mallimo** [comment]
> fallback externals do not include integrity values
But shouldn't they?

**Comment by JAdshead** [comment]
> > fallback externals do not include integrity values
> 
> But shouldn't they?

Yes, but instead of crashing in the client this will more gracefully load the script

File: package.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
     "fastify-plugin": "^4.5.1",
     "flat": "^5.0.2",
     "helmet": "^7.0.0",
-    "holocron": "^1.8.2",
```
Comment (JAdshead) [inline package.json:115]: This will need to be updated to next release with holocron updates

**Review by dogpatch626 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -112,7 +112,7 @@
...
+    "holocron": "^1.9.1",
```
Comment (Matthew-Mallimo) [inline package.json:115]: I this required? If I remember correctly the 1.9.0 release was done unintentionally?

Comment (JAdshead) [reply package.json:115]: yes, it includes the fixes for integrity loading in holocron

**Review by Matthew-Mallimo — APPROVED** [review]

**Review by dogpatch626 — APPROVED** [review]

**Pull request merged**